v2/internal/project/project.go

```diff
@@ -36,7 +37,7 @@ type Project struct {
 	/*** Internal Data ***/
 
 	// The path to the project directory
-	Path string
+	Path string `json:"projectdir"`
```

Comment:
We are now exposing this to the user, what's the meaning of this folder? AFAIK it's the folder where the `main.go` could be found, should we use a better naming for this?
Should this also be relative to the location of `wails.json` like frontendir and wailsjsdir? This would make it possible to put wails.json in the root of a project and put main.go of wails in e.g. `cmd/desktop`.

Reply:
Yeah, this looks like it was left over from a previous branch I cherry picked :/ I'll take this out.

Reply:
Great, thanks for clarifying. So then some of my comments will become obsolete 😀